src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -22,6 +23,139 @@
 
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
+
+    internal static class ClaimTypeMapping
```

Comment:
can we not have two classes in this file?

Reply:
good point, moved to its own